recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+import os, platform
+
+
+class ZuluOpenJDK(ConanFile):
+    name = "zulu-openjdk"
+    url = "https://github.com/conan-io/conan-center-index/"
+    description = "A OpenJDK distribution"
+    homepage = "https://www.azul.com"
+    license = "https://www.azul.com/products/zulu-and-zulu-enterprise/zulu-terms-of-use/"
+    topics = ("java", "jdk")
+    settings = "os", "arch"
+
```

Comment:
```suggestion

    @property
    def _source_subfolder(self):
        return "source_subfolder"

```

Use source_subfolder for source folder. Not because Conan needs, but because is a pattern in CCI.